app/templates/signup.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+

```

Comment:
Make a second file, named registerEmail.html, or anything kind of like that and put this content. Use Signup.html to get a username and a password (have two the password fields so they can be compared)